src/EnergyPlus/PipeHeatTransfer.cc

```diff
@@ -1882,7 +1879,7 @@ Real64 PipeHTData::OutsidePipeHeatTransCoef(EnergyPlusData &state)
     {
         auto const SELECT_CASE_var(this->TypeOf);
```

Comment:
`switch`/`case`.

Reply:
Since this is just comparing against the two enums, `DataPlant::PlantEquipmentType::PipeInterior` and `DataPlant::PlantEquipmentType::PipeExterior`, our preference is to use `BITF` over `switch/case` here, right? Referring to a previous discussion we had ([here](https://github.com/NREL/EnergyPlus/pull/8706#discussion_r612616667)). And as per that discussion, we should not have any more `else if` statements for `enum` class comparisons in the code, right?

Reply:
`BITF` is only for combining multiple sub-conditions in the same `if` state. If each condition is it's own `if` statement, then use switch/case or `if-else` if the number of conditions is small. Maybe in this case it's fine to keep `if-else` but just remove the `SELECT_CASE_var` wrapper. And the `Of`. :grin:

Reply:
Got it, Thank you :)